src/edit/tools/L.tools.EXIF.js

```diff
@@ -0,0 +1,62 @@
+L.tools = L.tools || {};
+
+L.tools.EXIF = function getEXIFdata(ref, overlay) {
+  var GPS = ref.exifdata,
```

Comment:
we would want to update this part to use 3 vars as per new code standardization rules